Mono.Addins.Gui/Mono.Addins.Gui/AddinManagerDialog.cs

```diff
@@ -183,12 +183,6 @@ void InsertFilterEntry ()
 			};
 			RepositionFilter ();
 		}
-
-		protected override void OnShown ()
-		{
-			base.OnShown ();
-			filterEntry.Parent = notebook;
-		}
```

Comment:
Why remove this?

Reply:
See 
https://github.com/mono/mono-addins/pull/138/commits/3a6c2d8f3880c4150511dda7d81cc020a55c10f4

Reply:
The `filterEntry` already has a parent, set when it is created